### DISCUSSION TITLE
谁知道长按词条误触忘记了如何恢复？

### DISCUSSION THREAD

**Discussion by JUST-INTJ** [fcitx5-android/fcitx5-android#477] [Q&A | 问答]
如图：
![Screenshot_20240423_115225_Via](https://github.com/fcitx5-android/fcitx5-android/assets/43703941/0c36b9fe-b35b-476a-a2d7-5e1c475de370)

**Comment by rocka** [comment]
应该不能直接恢复。如果你用的码表开启了学习功能，而且这个词条是符合码表组词规则的，可以用打单字的方法重复输入几遍，应该就能记住了。
